resources/UserData.ts

```diff
@@ -0,0 +1,22 @@
+
+export class UserData { 
+    private _email: string = "";
```

Comment:
Let's move away from using underscores for private properties, please.

Reply:
Yes, I'm sorry, this is from before you told me not to use them, they'll be gone in the next commit